### DISCUSSION TITLE
Page view spike in the last minute for Realtime on All Sites

### DISCUSSION THREAD

**Discussion by nfn** [umami-software/umami#1136] [General]
Hi,

I have multiple sites sharing the same installation and whenever I go to Realtime for All Sites I get a huge spike in the last minute:

16:10: https://i.ibb.co/yVrdxZb/1.png

16:16: https://i.ibb.co/G2QD1XG/2.png

16:20: https://i.ibb.co/5jPpLRQ/3.png

The values per site are ok, only when seeing All Sites I get this.
I'm using 1.25.0 installed from source.

Thanks

**Comment by mikecao** [comment]
I've never seen that before. Are you saying if you go through each website, none have stats that high?

**Comment by nfn** [comment]
None of the sites had spikes that justify this.
The strange thing is the 10:10 spike goes away when I refresh the page at 16:16 and the 10:16 spike goes away when I refresh the page at 10:20.

I'm monitoring this at the moment and I'll update this issue.
These sikes are not regular. The last days the behavior was correct.
